cg/services/delivery_message/messages/rna_fastq_scout_message.py

```diff
@@ -0,0 +1,60 @@
+from cg.services.delivery_message.messages.delivery_message import DeliveryMessage
+from cg.services.delivery_message.messages.utils import (
+    get_caesar_delivery_path,
+    get_scout_link,
+)
+from cg.store.models import Case
+from cg.store.store import Store
+
+
+class RNAFastqScoutMessage(DeliveryMessage):
+    def __init__(self, store):
+        super().__init__()
+        self.store = store
+
+    def create_message(self, cases: list[Case]) -> str:
+        if len(cases) == 1:
+            return self.get_case_message(cases[0])
+        return self.get_cases_message(cases)
+
+    def get_case_message(self, case: Case) -> str:
+
+        related_uploaded_dna_cases: list[Case] = self.store.get_uploaded_related_dna_cases(
+            rna_case=case
+        )
+        scout_links: list[str] = [get_scout_link(case) for case in related_uploaded_dna_cases]
+        scout_links_row_separated: str = "\n".join(scout_links)
+
+        delivery_path: str = get_caesar_delivery_path(case)
+        return (
+            f"Hello,\n\n"
+            f"The analysis for case {case.name} has been uploaded to the corresponding DNA case(s) on Scout at:\n\n"
+            f"{scout_links_row_separated}\n\n"
+            f"The analysis files are currently being uploaded to your inbox on Caesar:\n\n"
```

Comment:
should this be only fastq files instead of analysis files?

Reply:
Yes! Thanks! Changed now.